src/processors/snapshot-processor.ts

```diff
@@ -0,0 +1,10 @@
+// https://eslint.org/docs/developer-guide/working-with-plugins#processors-in-plugins
+
+type PostprocessMessage = { ruleId: string };
+
+export const snapshotProcessor = {
```

Comment:
`export default`?

Reply:
I've sort of started to [be against them](https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/). Not strongly, but like it's a think I'm trying and have started to kind of liked it and now encourage?

But I'm not married to it, so happy to make it a default export if you'd prefer :)

Reply:
In this case I'd ether go for 2 separate named exports (sa it currently is) or a single `default` one